roles/devscripts/tasks/sub_tasks/14_user.yml

```diff
@@ -44,3 +45,14 @@
     mode: "0640"
     content: |
       {{ cifmw_devscripts_user }}    ALL=(ALL)    NOPASSWD: ALL
+
+- name: Enable IP forwarding in the Network Operator.
+  when:
+    - cifmw_devscripts_config['network_type'] == 'OVNKubernetes'
+    - cifmw_devscripts_config['openshift_version'] is match("4\.14.*")
```

Comment:
what about future versions i.e 4.15+ ?

Reply:
Since we are seeing the problem with `4.14`, I have added the specific check

Reply:
I'd use the `is version()` test instead:
https://docs.ansible.com/ansible/latest/collections/ansible/builtin/version_test.html

Reply:
With https://issues.redhat.com/browse/OCPBUGS-3176 it changed default behavior so should apply to future versions, so would be good to cover those cases.

Reply:
Thank you @cjeanner ... something new. :)
Thanks @karelyatin , I agree that the check should be 4.14 and +

Made the necessary modifications in the next commit.